packages/grafana-ui/src/components/SingleStatShared/SingleStatBaseOptions.ts

```diff
@@ -202,6 +202,15 @@ export function sharedSingleStatMigrationHandler(panel: PanelModel<SingleStatBas
     delete options.fieldOptions;
   }
 
+  if (previousVersion < 7.1) {
+    // move title to displayName
+    const oldTitle = (panel.fieldConfig.defaults as any).title;
+    if (oldTitle !== undefined && oldTitle !== null) {
+      panel.fieldConfig.defaults.displayName = oldTitle;
+      delete (panel.fieldConfig.defaults as any).title;
+    }
+  }
+
```

Comment:
Shouldn't we handle this in the dashboard migrator? What if someone created i.e. Table panel in beta, with stable it's gonna no longer work.

Reply:
It’s going to be messy to do in dashboard migrator as that runs before any panel migration so would have to take all previous field config structures in mind

Reply:
Why would it not work for users running beta to stable? This migration will run for them as well

Reply:
yes, but they will have to switch to stat-ish viz first right? But nvm, it's still gonna work as we are not adding nor removing any fields, so all good. Thinks this is good to merge! :)